securitycenter/google/cloud/securitycenter_v1beta1/types.py

```diff
@@ -24,7 +24,8 @@
 from google.cloud.securitycenter_v1beta1.proto import finding_pb2
 from google.cloud.securitycenter_v1beta1.proto import organization_settings_pb2
 from google.cloud.securitycenter_v1beta1.proto import security_marks_pb2
-from google.cloud.securitycenter_v1beta1.proto import securitycenter_service_pb2
+from google.cloud.securitycenter_v1beta1.proto \
+    import securitycenter_service_pb2
```

Comment:
Hmm, did you fix this one manually?  We normally exclude the generated code from the linter, if that was the motive (but it looks like you did that above?)

Reply:
fixed.